opendebates/views.py

```diff
@@ -352,6 +352,16 @@ def questions(request):
     return redirect(url + "#created=%s" % idea.id)
 
 
+@rendered_with("opendebates/changelog.html")
+def changelog(request):
+    moderated = Submission.objects.filter(
+        Q(approved=False) | Q(duplicate_of__isnull=False)
+    ).select_related('duplicate_of').order_by('id')
```

Comment:
Is the ascending order here intentional?  I would've assumed "most recent first".